### PR TITLE
UID-154 New utility in developer settings for looking up capabilities/capabilitySets by permission name

### PR DESCRIPTION
**Pull request by ryandberger**
- Fulfills [UID-154](https://folio-org.atlassian.net/browse/UID-154)
- Allows user to query capabilities and capabilitySets by capability name
- [UID-156](https://folio-org.atlassian.net/browse/UID-156) will add searching by capability display name

https://github.com/folio-org/ui-developer/assets/28395235/7eeef1cc-b232-4790-acc4-f20a01e51f93

### CODE REVIEW

File: src/settings/ShowCapabilities.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      <ul>
+        <li>{value.permission}</li>
+        <ul>
+          <li><strong>type:</strong> {value.type}</li>
+          <li><strong>applicationId:</strong> {value.applicationId}</li>
+          <li><strong>resource:</strong> {value.resource}</li>
+          <li><strong>action:</strong> {value.action}</li>
+          <li><strong>permissionName:</strong> {value.name}</li>
+          <li><strong>permissionDisplayName:</strong> {value.description}</li>
+        </ul>
```
Comment (zburke) [inline src/settings/ShowCapabilities.js:38]: `<ul>` is fine, but `* key: value` smells a lot like a `<dl>`.

File: src/settings/ShowCapabilities.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+import { Button, Pane, Row, Col, SearchField } from '@folio/stripes/components';
+
+const ShowCapabilities = () => {
+  const ky = useOkapiKy();
+  const [capabilitiesResults, setCapabilitiesResults] = useState({});
+  const [capabilitySetsResults, setCapabilitySetsResults] = useState({});
+  const [query, setQuery] = useState('');
+
+  const searchParams = {
+    limit: 1000,
```
Comment (zburke) [inline src/settings/ShowCapabilities.js:14]: Can you pull a limit from stripes-config instead -- don't we have a constant there, or in stripes-core somewhere? Magic numbers make me twitchy.

**Review by zburke — APPROVED** [review]
"I can haz capabilities" So glad we are maintaining parity with existing jokes 😆 

LGTM

**Review by aidynoJ — APPROVED** [review]

File: src/settings/ShowCapabilities.js (hunk 1 above)
Comment (ryandberger) [reply src/settings/ShowCapabilities.js:38]: Seems like `stripes-components` `List` uses `<li>`. The formatting per the screenshot seems to work better with `<ul>` in this case.

**Pull request merged**